modules/component/src/push/push.pipe.ts

```diff
@@ -67,63 +54,46 @@ import {
  */
 @Pipe({ name: 'ngrxPush', pure: false })
 export class PushPipe<S> implements PipeTransform, OnDestroy {
-  private renderedValue: S | null | undefined;
-
-  private readonly configSubject = new Subject<PushPipeConfig>();
-  private readonly config$ = this.configSubject
-    .asObservable()
-    .pipe(distinctUntilChanged());
+  private renderedValue: any | null | undefined;
```

Comment:
`any | null | undefined` is just `any`. Maybe be more specific?

Reply:
Done